fixtures/hsl/40w-beam-spot-light-rgbw.json

```diff
@@ -0,0 +1,123 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "40W Beam Spot Light RGBW",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["David Oleniacz"],
+    "createDate": "2023-04-19",
+    "lastModifyDate": "2023-04-19"
+  },
+  "links": {
+    "productPage": [
+      "https://www.amazon.com/stores/HighIightSpIendidLife/page/C6F4337A-41CC-4560-880E-06BCA9E96924"
+    ]
+  },
```

Comment:
Missing link to manual and video. Cannot validate fixture all aspects without the manual. The DMX description can be found in [this Amazon Q&A](https://www.amazon.com/ask/questions/Tx3CE66RTIHOGZL/ref=ask_dp_dpmw_al_hza)